src/Sarif/PropertiesDictionaryExtensionMethods.cs

```diff
@@ -73,33 +74,28 @@ public static class PropertiesDictionaryExtensionMethods
                 StringSet stringSet = property as StringSet;
                 if (stringSet != null)
                 {
+                    SaveSettingNameToDescriptionMap(writer, key, settingNameToDescriptionMap);
```

Comment:
>SaveSettingNameToDescriptionMap [](http://example.com/codeflow?start=20&length=31)

pls, add tests that cover this change.

Reply:
found this new method is not needed, removed.
the bug was comment not written to file,
the method I add was to test that. PropertiesDictionary_TestCacheDescription